Source/dafny.msbuild/build/dafny.msbuild.targets

```diff
@@ -3,8 +3,8 @@
     <Target Name="VerifyDafny">
         <DafnyVerifyTask DafnySourceFiles="@(DafnySource)" 
                          DafnyExecutable="$(DafnyBinariesDir)$(Dafny)"
-                         TimeLimit="$(DafnyVerificationTimeLimit)"
-                         Jobs="$(VerifyDafnyJobs)"/>
+                         Jobs="$(VerifyDafnyJobs)"
+                         DafnyVerificationParams="@(VerifyDafnyPassthrough);$(VerifyDafnyPassthrough)" />
```

Comment:
I still want to understand the weird syntax here. I suspect the $() one is always blank?

Reply:
Sorry, missed your comment above, let me revisit.

Reply:
I updated this on: https://github.com/dafny-lang/dafny.msbuild/pull/11/commits/31c1ee6cd3ca25b65951679f01a0a92a5a91523f

We can always override this in a csproj of our own by passing a CLI param (which could be a `;` separated list into an ItemGroup. The param name can be the same as the itemgroup `VerifyDafnyPassthrough` or it could be something different. We're customizing everything ourselves anyways, so this also allows us to change the CLI name if we want.